packages/core/src/renderer/Renderer.ts

```diff
@@ -40,74 +38,26 @@ export interface RenderProps {
   texLabels: boolean;
   canvasSize: [number, number];
   pathResolver: PathResolver;
+  titleCache?: Map<string, SVGElement>;
 }
-export type InteractiveProps = {
-  updateState: (newState: State) => void;
-  onDrag: (id: string, dx: number, dy: number) => void;
-  parentSVG: SVGSVGElement;
-};
+
 /**
  * Converts screen to relative SVG coords
  * Thanks to
- * https://www.petercollingridge.co.uk/tutorials/svg/interactive/dragging/
+ * https://www.petercollingridge.co.uk/tutorials/svg/interxactive/dragging/
  * @param e
- * @param svg
+ * @param CTM
  */
 const getPosition = (
-  { clientX, clientY }: { clientX: number; clientY: number },
-  svg: SVGSVGElement,
+  { screenX, screenY }: { screenX: number; screenY: number },
+  CTM: DOMMatrix | null,
 ) => {
-  const CTM = svg.getScreenCTM();
   if (CTM !== null) {
-    return { x: (clientX - CTM.e) / CTM.a, y: (clientY - CTM.f) / CTM.d };
+    return { x: (screenX - CTM.e) / CTM.a, y: (screenY - CTM.f) / CTM.d };
   }
   return { x: 0, y: 0 };
 };
```

Comment:
looks like this function is no longer used?

Reply:
Yep, good catch.